x-pack/packages/kbn-elastic-assistant/impl/assistant/use_assistant_overlay/index.tsx

```diff
@@ -104,16 +130,52 @@ export const useAssistantOverlay = (
   // proxy show / hide calls to assistant context, using our internal prompt context id:
   // silent:boolean doesn't show the toast notification if the conversation is not found
   const showAssistantOverlay = useCallback(
-    async (showOverlay: boolean) => {
+    // shouldCreateConversation should only be passed for
+    // non-default conversations that may need to be initialized
+    async (showOverlay: boolean, shouldCreateConversation: boolean = false) => {
       if (promptContextId != null) {
+        if (shouldCreateConversation) {
+          let conversation;
+          if (!isLoading) {
+            conversation = conversationTitle
+              ? Object.values(conversations).find((conv) => conv.title === conversationTitle)
+              : undefined;
+          }
+
+          if (isAssistantEnabled && !conversation && defaultConnector && !isLoading) {
+            try {
+              await createConversation({
+                apiConfig: {
+                  ...apiConfig,
+                  actionTypeId: defaultConnector?.actionTypeId,
+                  connectorId: defaultConnector?.id,
+                },
+                category: 'assistant',
+                title: conversationTitle ?? '',
+              });
+            } catch (e) {
+              /* empty */
+            }
```

Comment:
Was curious about the error behavior here so tested a bit and all looks good. 👍  If thrown, `createConversation` will throw up a toast, and the assistant flyout will open to the `lastConversationId` with the AttackDiscovery context available.